docs/pests.md

```diff
@@ -33,34 +32,38 @@ pest:
     value: 0.05
 ```
 
-The `random` arrangement does not have any further configuration besides
-`infestation_rate`.
+## Pest arrangement
+The pest arrangement within the shipment can be determined by:
+
+```
+pest:
+  arrangement: random
+```
+
+The `arrangement` can be `random` for uniform random distribution,
+`clustered` for one or more pest clusters within stems of the shipment,
+and `random_box` for one or more random boxes having pest.
+
+### Random arrangement
+The `random` arrangement does not have any further configuration. The computed number of stems or boxes (based on the infestation rate described above) are uniform randomly selected from the total number of units and infested.
 
-The settings for `clustered` is:
+### Clustered arrangement
+If using `arrangement` = `clustered`, the configuration is as follows:
 
 ```
   clustered:
-    max_stems_per_cluster: 10
+    max_infested_stems_per_cluster: 10
     distribution: gamma
     parameters:
     - 4
     - 2
 ```
 
-The cluster size in terms of number of stems is limited by
-`max_stems_per_cluster`. If `max_stems_per_cluster` is exceeded, more
-than one cluster is generated so that number of stems in each cluster
-conforms to this limit.
+The cluster size in terms of infested stems is limited by `max_infested_stems_per_cluster`. If `max_infested_stems_per_cluster` is exceeded, more than one cluster is generated so that number of stems in each cluster conforms to this limit. Note that `max_infested_stems_per_cluster` is also used to determine infested boxes per cluster when `infestation_unit` = `boxes` by dividing `max_infested_stems_per_cluster` by `stems_per_box`.
```

Comment:
Why not `max_units_stems_per_cluster` instead of `max_infested_stems_per_cluster`. We use `...unit[s]...` elsewhere, it does not seem to make a difference in the code, and it would remove the need to get `stems_per_box` from shipment generation config (which is the part I'm interested in). It would simplify the doc, too.

Reply:
Yes, I think that makes sense, but I'd like to wait to make this change in the next branch I have started for cleaning up some things, including removing the gamma clustering option and renaming the one parameter for random clustering to max_cluster_width. That way we can see how the variable name changes will work together.